### PR TITLE
Feature/select popover

### PR DESCRIPTION
**Pull request by zhakhalov**
#### Short description of what this resolves:

Added new type of interfaces to ion-select component. New 'popover' type of interface gives single-option input of HTML select using ion-popover component. Good solution for desktop apllications and large Tablets.

---
##### iOS preview:

screen shot 2016-08-24 at 5 58 57 pm

---
##### MaterialDesign Preview:

screen shot 2016-08-24 at 5 57 02 pm
#### Changes proposed in this pull request:
- new type(popover) of select interface added
- demos for `interface="popover"` added

**Ionic Version**: 2.x

### CODE REVIEW

File: src/components/select/select.ios.scss [hunk 1]
```diff
@@ -12,6 +12,11 @@ $select-ios-padding-left:        $item-ios-padding-left !default;
 $select-ios-icon-color:          #999 !default;
 $select-ios-placeholder-color:   $select-ios-icon-color !default;
 
+$select-popover-ios-margin-top:         -1px !default;
+$select-popover-ios-margin-right:       0 !default;
+$select-popover-ios-margin-bottom:      -1px !default;
+$select-popover-ios-margin-left:        0 !default;
```
Comment (brandyscarney) [inline src/components/select/select.ios.scss]: Could these be changed to start with `$select-ios-popover-`? I like to use the following order for naming variables:

```
<component>-<mode>-<child element or component>-<css property>
```

See the following document for more info: https://docs.google.com/document/d/1OyOyrRE5lpB_9mdkF0HWVQLV97fHma450N8XqE4mjZQ/edit

Thanks :)

File: src/components/select/select-popover-component.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { Component, OnInit } from '@angular/core';
+import { NavParams } from '../nav/nav-params';
+import { ViewController } from '../nav/view-controller';
+
+/** @private */
+export interface ISelectPopoverOption {
```
Comment (brandyscarney) [inline src/components/select/select-popover-component.ts]: Could you remove the `I` from the beginning of the interface name please? :)

**Comment by brandyscarney** [comment]
Thanks for the PR! This is awesome! I added some comments on a couple lines. Let me know if you have questions! :smile:

**Comment by m0n01i7h** [comment]
Wow!
So fast response :)
I've updated PR due to comments.

**Comment by m0n01i7h** [comment]
Hi,
Any updates?

**Comment by mifkys** [comment]
Version for Windows 10? (:

**Comment by BugsBunnyBR** [comment]
@m0n01i7h , could you update your `PR` ? Maybe the Ionic team can merge it before 2.0 release.

**Comment by m0n01i7h** [comment]
Hi, 
I have updated my PR

**Comment by brandyscarney** [comment]
Just to update on this - we are aware and appreciative of this PR and we do want to merge it in the future. However, we are focused on fixing existing bugs before implementing enhancements. Hope you understand! :)

**Comment by danbucholtz** [comment]
@zhakhalov, This is hot 🔥 🔥 🔥 🌶 🌶 🌶

**Comment by zhakhalov** [comment]
Are you going to merge it?

**Comment by brandyscarney** [comment]
@zhakhalov I'm working on it. I had to fix some conflicts. :)

**Comment by brandyscarney** [comment]
Thanks a ton for this PR, @zhakhalov! Sorry it took so long to get it in. I had to make some changes to get it working with master, but it has been merged in this commit: https://github.com/driftyco/ionic/commit/745d80872831efebe958a18789d662da60a4ba37

A nightly has been released with it and can be tested by running:

```
npm install --save --save-exact ionic-angular@3.0.1-201704172046
```

And it will be included in the `3.1.0` release. Thanks again, this is great! :tada:

**Pull request closed without merging**